README.md

```diff
@@ -68,6 +68,12 @@ their impact. We support both Intel and Apple Silicon macs now and contain this
 `ThermalsParser.kt`. This also includes helpful charting APIs for visualizing the data (courtesy
 of our friends at Square).
 
+### Better Properties
+
+Gradle's built-in property support is limited and has surprising behavior, so we have our own 
+system on top of it that has consistent precedence, `local.properties` support, local project 
+properties, and configuration caching support. Check out `safeProperty()` in `PropertyUtil.kt`.
```

Comment:
Minor nit - `local.properties` support vs. local project properties. 

We explicitly mention `local.properties`, which makes sense, but I think “local project properties” includes `local.properties`.

Perhaps we can expand on the latter more or delete the former?

Reply:
nope they are different things. One comes from `local.properties` files and the other comes from project-local `gradle.properties` files

Reply:
Thanks that makes sense. We can elaborate the latter with your response here with "project-local `gradle.properties` files", but this was just a nit.